pkg/controller/controller.go

```diff
@@ -489,21 +503,30 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) {
 		if instance.Spec.Parameters != nil {
 			parameters, err = unmarshalParameters(instance.Spec.Parameters.Raw)
 			if err != nil {
-				glog.Errorf("Failed to unmarshal Instance parameters\n%s\n %v", instance.Spec.Parameters, err)
+				s := fmt.Sprintf("Failed to unmarshal Instance parameters\n%s\n %s", instance.Spec.Parameters, err)
+				glog.Info(s)
 				c.updateInstanceCondition(
 					instance,
 					v1alpha1.InstanceConditionReady,
 					v1alpha1.ConditionFalse,
 					errorWithParameters,
-					"Error unmarshaling instance parameters",
+					"Error unmarshaling instance parameters. "+s,
 				)
 				return
 			}
 		}
 
 		ns, err := c.kubeClient.Core().Namespaces().Get(instance.Namespace, metav1.GetOptions{})
 		if err != nil {
-			glog.Errorf("Failed to get namespace during instance create (%s)", err)
+			s := fmt.Sprintf("Failed to get namespace %q during instance create (%s)", instance.Namespace, err)
```

Comment:
If you want. I was copying the existing formatting. There are a lot of `(%%)` strings around.